src/plugins/janus_audiobridge.c

```diff
@@ -8572,9 +8574,53 @@ static void *janus_audiobridge_participant_thread(void *data) {
 			if(participant->jitter) {
 				ret = jitter_buffer_get(participant->jitter, &jbp, participant->codec == JANUS_AUDIOCODEC_OPUS ? 960 : 160, NULL);
 				jitter_buffer_tick(participant->jitter);
-				if(ret != JITTER_BUFFER_OK) {
-					/* No packet in the jitter buffer? Move on the talking detection, if needed */
+
+				if(!first && participant->codec == JANUS_AUDIOCODEC_OPUS && (ret == JITTER_BUFFER_MISSING || ret == JITTER_BUFFER_INSERTION) && lost_packets_gap <= JITTER_BUFFER_MAX_GAP_SIZE) {
```

Comment:
The previous code invoked `janus_audiobridge_participant_istalking` regardless of any other condition than `ret != JITTER_BUFFER_OK`, please restore this beahvior.

Reply:
this is left on line 8577, jitter_buffer_get doesn't return anything other than JITTER_BUFFER_MISSING || JITTER_bUFFER_INSERTION || JITTER_BUFFER_OK

Reply:
It's not the same.
The old code does this:
```
// every 20 ms loop
ret = jitter_buffer_get(participant->jitter, &jbp, participant->codec == JANUS_AUDIOCODEC_OPUS ? 960 : 160, NULL);
jitter_buffer_tick(participant->jitter);
if(ret != JITTER_BUFFER_OK) {
	janus_audiobridge_participant_istalking(session, participant, NULL, NULL);
} else {
	//parse g711 or decode opus (with fec)
	janus_audiobridge_participant_istalking
}
```

Your patch does not cover some paths, for example a g711 packet goes always in the decode path -> WRONG, should just do `janus_audiobridge_participant_istalking`.

IMHO this should be written as:

```
if(ret != JITTER_BUFFER_OK) {
	if(!first && participant->codec == JANUS_AUDIOCODEC_OPUS && lost_packets_gap <= JITTER_BUFFER_MAX_GAP_SIZE) {
		// Opus PLC stuff
                ...
	}
	else {
		/* No packet in the jitter buffer? Move on the talking detection, if needed */
		janus_audiobridge_participant_istalking(session, participant, NULL, NULL);
	}
} else {
	/* Decode the audio packet */
        ....
}
```

Reply:
ok, changed